rtos/TARGET_CORTEX/rtx5/RTX/Source/TOOLCHAIN_GCC/TARGET_M3/irq_cm3.S

```diff
@@ -76,15 +76,6 @@ SVC_ContextSwitch:
 #ifdef FEATURE_UVISOR
         CPSID I // The call to the thread switch helper and PSP loading must be atomic.
 #endif
-        /* The call to thread_switch_helper can clobber R2 and R3, but we don't
```

Comment:
and this block too

Reply:
@bulislaw FYI, these come from the commit "RTX5: uVisor: Switch threads very carefully" (778d6822bff9013f22793aee41a891a440c16108).